solutions/silver/cf-862B.mdx

```diff
@@ -115,5 +115,48 @@ int main() {
 }
 ```
 </CPPSection>
+<PySection>
 
+```py
+MAX_N = 100000
+
+# +1 for 1-indexed nodes
+adj = [[] for _ in range(MAX_N + 1)]
```

Comment:
can't you size this dynamically?

Reply:
For adjacency list I thought it was standard to use for _ in range. How would that work dynamically? I think just making it adj = [] will give RE.

Reply:
like define it based off `n`

Reply:
oh yeah, you're right